pallets/schemas/src/rpc/src/lib.rs

```diff
@@ -1,17 +1,19 @@
-use codec::Codec;
-use core::result::Result as CoreResult;
-use jsonrpc_core::{Error as RpcError, ErrorCode, Result};
+use common_primitives::{rpc::*, schema::*};
+use jsonrpc_core::Result;
 use jsonrpc_derive::rpc;
 use pallet_schemas_runtime_api::SchemasRuntimeApi;
-use sp_api::{ApiError, ProvideRuntimeApi};
+use sp_api::ProvideRuntimeApi;
 use sp_blockchain::HeaderBackend;
-use sp_runtime::{generic::BlockId, traits::Block as BlockT, DispatchError};
+use sp_runtime::{generic::BlockId, traits::Block as BlockT};
 use std::sync::Arc;
 
 #[rpc]
-pub trait SchemasApi<BlockHash, AccountId> {
+pub trait SchemasApi<BlockHash> {
```

Comment:
Do we need BlockHash? Maybe I did not catch how we are using it.

Reply:
it is used in previously written rpc `get_latest_schema_id`

Reply:
We want to avoid to be able to query by blockhash right?